Sources/TuistGraph/DependenciesGraph/DependenciesGraph.swift

```diff
@@ -14,3 +14,37 @@ public struct DependenciesGraph: Equatable, Codable {
         self.thirdPartyDependencies = thirdPartyDependencies
     }
 }
+
+public enum DependenciesGraphError: FatalError, Equatable {
```

Comment:
We have a convention for the order of definitions in a file:

```swift
// MARK: - AbcError

enum AbcError: Error {
    //
}

// MARK: - AbcProtocol

protocol AbcProtocol {
    //
}

// MARK: - Abc

class Abc: AbcProtocol {
    //
}
```

Examples:
- [here](https://github.com/tuist/tuist/blob/main/Sources/TuistCore/Utils/BinaryLocator.swift)
- [here](https://github.com/tuist/tuist/blob/main/Sources/TuistDependencies/Carthage/CarthageInteractor.swift)
- [here](https://github.com/tuist/tuist/blob/main/Sources/TuistGenerator/Generator/WorkspaceDescriptorGenerator.swift)

Id like to suggest to follow the convention here as well.